lib/sles4sap_publiccloud.pm

```diff
@@ -182,6 +184,70 @@ sub sles4sap_cleanup {
 
 }
 
+=head2 get_job_state
+    Get whether a specified job is still running or not
+=cut
+
+sub get_job_state {
+    my ($job_id) = @_;
+
+    my $url = "https://openqa.suse.de/api/v1/jobs/$job_id";
```

Comment:
is it possible not hardcoding the host name here?
Is it the right place for a such generic function?

Reply:
is there no such setting already available?

Reply:
Use **OPENQA_HOSTNAME**

Reply:
Done